src/util/serialize.h

```diff
@@ -253,26 +253,34 @@ inline v3s32 readV3S32(const u8 *data)
 inline v2f readV2F1000(const u8 *data)
 {
 	v2f p;
-	p.X = (float)readF1000(&data[0]);
-	p.Y = (float)readF1000(&data[4]);
+	p.X = readF1000(&data[0]);
+	p.Y = readF1000(&data[4]);
 	return p;
 }
 
 inline v3f readV3F1000(const u8 *data)
```

Comment:
is this always used ?

Reply:
It is used for map-data or any serialization which is persistent (save to disk).